### ISSUE TITLE
Mirrored display

### ISSUE THREAD

**Issue by 2BWorks** [adamkovesdi/esp-matrixclock#4]
Hey Thanks for this awesome project, i have managed to compile the code and the display is displaying something although it appears to be mirrored. I'm using cheap AliExpress 4 tile matrix 8x8 leds.

**Comment by adamkovesdi** [comment]
Take a look into the display code (max7219.cpp). [source link](https://github.com/adamkovesdi/esp-matrixclock/blob/40d3150f33a9d775e9941fd749dae7fc4a6d242f/src/max7219.cpp#L71)
If you use the 'old type' matrix PCB, there is this definition for mirrored output: OLDTYPE_MATRIX

I had 2 different matrices and each of them needed different code. Reason behind this is the vendor of the matrix display changed chaining of the individual 8x8 blocks.

TL;DR define OLDTYPE_MATRIX in max7219.cpp and see if that helps.
Add the following line in max7219.h:
`#define OLDTYPE_MATRIX`

**Comment by AivarJ** [comment]
Same problem as mentioned above and got its working after adding following line in max7219.h 

#define OLDTYPE_MATRIX.

Thanks.

**Comment by adamkovesdi** [comment]
Thanks for confirming. Closing this.
